manifests/0000_30_cluster-baremetal-operator_04_deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-baremetal-operator
+  namespace: openshift-machine-api
+  labels:
+    k8s-app: cluster-baremetal-operator
+  annotations:
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: cluster-baremetal-operator
+  template:
+    metadata:
+      labels:
+        k8s-app: cluster-baremetal-operator
+    spec:
+      priorityClassName: system-node-critical
+      serviceAccountName: cluster-baremetal-operator
+      containers:
+      - name: cluster-baremetal-operator
```

Comment:
The container spec is going to need an image to be valid.

What would it take to get a quay.io image for the CBO, so we could use these files to test deploying it ourselves locally?

Reply:
That's a point I wanted to clarify (the image references), I will follow up on this topic

Reply:
Ok, I think that this manifest added by @sadasu will fix the problem: https://github.com/openshift/cluster-baremetal-operator/blob/f2f3f9e213becd309d973350052aaef39c434de9/manifests/image-references#L5